src/rules/no-identical-functions.ts

```diff
@@ -30,6 +30,7 @@ const message = (line: string) =>
 
 const rule: Rule.RuleModule = {
   meta: {
+    type: "problem",
```

Comment:
this rule mostly reposting code smells (code duplication) and not bad copy paste (see https://jira.sonarsource.com/browse/RSPEC-4144 is code smell). So I suggest to use `suggestion` here